src/GitHub.TeamFoundation.14/Services/TeamExplorerServices.cs

```diff
@@ -110,5 +123,37 @@ void OpenFolder(string repositoryPath)
             var dte = serviceProvider.TryGetService<DTE>();
             dte?.ExecuteCommand("File.OpenFolder", repositoryPath);
         }
+
+        static void BringIntoView(ITeamExplorerSection section)
+        {
+            var control = section?.SectionContent as UserControl;
+            if (control != null)
+            {
+                if (control.IsLoaded)
+                {
+                    BringIntoView();
+                }
+                else
+                {
+                    control.Loaded += (s, e) => BringIntoView();
```

Comment:
Shouldn't the event handler deregister itself before calling `BringIntoView()`?

Reply:
I've changed this to use Rx, which should automatically handle the un-subscription.